tests/js/server/dump/dump-test.inc

```diff
@@ -1184,7 +1185,136 @@
           assertTrue(next < 100000000, "expected next uniqid after restore to be less than 100000000, not " + next);
         }
       },
-      
+
+      ////////////////////////////////////////////////////////////////////////////////
+      /// @brief test smart graph restores
+      ////////////////////////////////////////////////////////////////////////////////
+      testEmptySmartGraph: function () {
+        const smartGraphName = "UnitTestsDumpEmptySmartGraph";
```

Comment:
The name is already defined in `tests/js/server/dump/dump-setup-common.inc` and should be passed here as a parameter.

Reply:
This and other tests added in the PR are special cases that benefit from qualifiers in their name, also to keep them unique.

The pre-defined name would not be suitable here.